Automattic/wp-calypso
# ETK Migration: Remove Block Editor Nav Sidebar

Related to pekYwv-4bZ-p2

## Proposed Changes

⚠️ This PR should be merged AFTER ETK including https://github.com/Automattic/wp-calypso/pull/92583 is shipped to both Simple and Atomic.

Removing Block Editor Nav Sidebar since it's no longer in use. See p1720587845196889-slack-CRWCHQGUB for more context. 

## Testing Instructions



* Patch this change to your sandbox with: https://github.com/Automattic/wp-calypso/pull/92581#issuecomment-2222167761
* Go to the Post Editor
	* Click the W icon to go back
	* See that nothing is broken and no error console logs
* Go to Site Editor
	* See that nothing is broken and no error console logs

## Pre-merge Checklist
